typhon/plugins/core.py

```diff
@@ -181,9 +184,13 @@ def add_connection(self, channel):
         try:
             # Add a PyDMConnection for the channel
             super().add_connection(channel)
-        # There is a chance that we raise an Exception on subscription. If so,
-        # don't add this to our list of good to go exceptions so the next
+        # There is a chance that we raise an Exception on creation. If so,
+        # don't add this to our list of good to go connections. The next
         # attempt we try again.
```

Comment:
What do you mean by: "The next attempt we try again?
If you have a widget asking to connect to a certain plugin and it fails I don't think that it will request the connection again... maybe we should add something at PyDM to retry? I'm not sure if I ever saw this case.
For Epics we add the listener anyway since we get a notification when the PV is connected... In this case, you are not adding the connection to the channels when it fails to subscribe...

Reply:
I mean, if you have two listeners request the same channel, the first one fails, we still will try again on the second i.e

```python
PyDMWidget(init_channel='sig://my_signal')  # Fails from network hiccup
PyDMWIdget(init_channel='sig://my_signal')  # Try again to create a channel
```